### PR TITLE
Wanda carlson patch 22

### PR DESCRIPTION
**Pull request by wanda-carlson**
```
<div>Hello world, and Henry</div>
```

image

### CODE REVIEW

**Comment by wanda-carlson** [comment]
This is a PR with skipped tests

**Review by henry-richard — CHANGES_REQUESTED** [review]
i dont like this

**Comment by henry-richard** [comment]
A little tough to understand

**Review by henry-richard — APPROVED** [review]

**Comment by wanda-carlson** [comment]
@henry-richard does this work?

File: src/bert.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+print("hello world")
+print("hello world")
+print("hello world")
```
Comment (henry-richard) [inline src/bert.py:3]: this is nice

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by henry-richard — APPROVED** [review]
It's ok

**Pull request merged**